php_phongo.c

```diff
@@ -2016,6 +2016,18 @@ static void php_phongo_client_dtor(void *client)
 }
 #endif
 
+static void _phongo_check_json (void) __attribute__((constructor));
+static void _phongo_check_json (void)
```

Comment:
Does `__attribute__((constructor))` result in this preempting PHP's own extension dependency resolution based on the list in our zend_module_entry?